app/pb_kits/playbook/pb_selectable_icon/_selectable_icon.scss

```diff
@@ -0,0 +1,85 @@
+[class^=pb_selectable_icon] {
+
+  text-align: center;
+  cursor: pointer;
+
+  :hover * {
+		filter: drop-shadow(0 0 8px rgba($shadow, 0.25));
+		transition-duration: $transition_short;
+		transition-timing-function: $easeIn;
+  }
+
+  * {
+    cursor: pointer;
+  }
+
+  h4 {
+    margin-top: $space_xs !important;
+    user-select: none;
+  }
+
+  &[class*=_disabled] {
+		opacity: $opacity_6;
+		cursor: not-allowed;
+
+		* {
+				cursor: not-allowed;
+		}
+	}
+	
+  input[type="checkbox"],
+  input[type="radio"] {
+		-webkit-appearance: none;
+		-moz-appearance: none;
+		display: none;
+		appearance: none;
+
+		transition-property: color;
+		transition-duration: $transition_short;
+		transition-timing-function: $easeIn;
+
+		&:checked ~ label * {
+			color: $primary;
+		}
+		
+		&:not(:checked) ~ label {
+			color: $slate;
+
+			h4 {
+				transition-delay: .15s;
+			}
+		}
+	}
+
+
+  &[class*=_dark] {
+
+		input[type="checkbox"],
+		input[type="radio"] {
+			-webkit-appearance: none;
+			-moz-appearance: none;
+			display: none;
+			appearance: none;
+
+			transition-property: color;
+			transition-duration: $transition_short;
+			transition-timing-function: $easeIn;
+
+			&:checked ~ label * {
+				color: $active_dark !important;
+			}
+				
+			&:not(:checked) ~ label {
+				
+				svg {
+					color: $slate;
+				}
```

Comment:
```suggestion
					color: $text_dk_lighter;
```

Reply:
Don't target the SVG. FA will color the icon based on the text properties you add to the parent